hw/dv/tools/vcs/xprop.cfg

```diff
@@ -0,0 +1,4 @@
+merge = tmerge;
+
+// Turn on xprop for dut only
+instance { tb.dut } { xpropOn };
```

Comment:
Should this be 'tree' (apply to all sub-hierarchies) instead of 'instance' (only apply to the indicated instance)?

Reply:
No, it applies to all sub-instance as well. If `tree`, need to use module name like (uart)

Reply:
Okay, thanks for confirming.